src/main/java/gr/cytech/chatreminderbot/rest/controlCases/CaseCancelReminder.java

```diff
@@ -21,7 +22,7 @@
 
     private List<String> splitMsg;
 
-    public CaseDeleteReminder() {
+    public CaseCancelReminder() {
     }
 
     public Request getRequest() {
```

Comment:
is it possible to remove these setters?
It would be best if you did not alter the class's state from the outside, only passing any required argument through the cosntructor, or directly through method parameters